packages/atlas/atlas.config.yml

```diff
@@ -174,6 +174,7 @@ content:
       name: Vietnamese
     - isoCode: yo
       name: Yoruba
+  popularLanguages: ['ar', 'zh', 'en', 'fr', 'de', 'ja', 'ms', 'pt', 'ru', 'st']
```

Comment:
I'm not sure if this list should be sorted alphabetically. I would display the english language channel first, then probably russian(since we have a lot of russian speaking users)

Reply:
I'm not sure about this, this is how it was ordered here: https://docs.google.com/spreadsheets/d/1DS8nyJWLQpXWKrX9q697VHj2snV4rUwTVkWN3PlKEOE/edit#gid=0

Let's confirm it with @dmtrjsg

Reply:
We agreed that we will sort them alphabetically except for English which will be in first place